style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: hsla(354, 90%, 47%, 0.504);
+}
```

Comment:
- [ ] As required for the project, kindly ensure that your project includes a professional README file and it should be descriptive. And for that make sure you use the  **[AKA: Microverse README template](https://github.com/microverseinc/readme-template)**. After you have added the README you should make sure all links inside are working and all main sections are present with the appropriate content related to your project.
*

Reply:
https://github.com/VivekRaaZ/Hello-Microverse/tree/html-css-Branch#readme

Reply:
Does this it right ?